bin/rialto/node/src/service.rs

```diff
@@ -117,8 +117,7 @@ type FullGrandpaLink = sc_finality_grandpa::LinkHalf<Block, FullClient, FullSele
 
 /// A set of APIs that polkadot-like runtimes must implement.
 ///
-/// This is the copy of `polkadot_service::RuntimeApiCollection` with some APIs removed
-/// (right now - MMR and BEEFY).
+/// This is a copy of `polkadot_service::RuntimeApiCollection`.
```

Comment:
Does it still need to be a copy now? I.e. could we re-use that module directly from polkadot?

Reply:
Used the polkadot upstream definition :+1: